WebRoot/skins/_base/base4/skin.css

```diff
@@ -1276,4 +1276,9 @@ div[name='_mergeBtnContainerId'] .ZCalToggleBtnSelected .ZWidgetTitle {
 /* Today Inline button for Calendar Print Dialog */
 .ZmCalPrintDialog td[id$="todayButtonContainer"] .ZWidgetTitle {
     @AlignLeft@
-}
\ No newline at end of file
+}
+
+.ZmApptQuickAddDialog .ZSelect .ZWidgetTitle {
+    max-width: 1px;
```

Comment:
having a max width of 1px, isn't this weird?

Reply:
Yes, it sounds so. But it's actually combination of `width:100%` and `max-width:1px` which effectively helps us to achieve `text-overflow: ellipsis` behaviour here. And it works for all major browsers. 

You can find more here: https://stackoverflow.com/questions/9789723/css-text-overflow-in-a-table-cell

Thanks